gen3/docs/gen3-resources/glossary.md

```diff
@@ -36,58 +36,45 @@ Edge nodes may be created at clinics, labs, hospitals, or academic institutions
-to a user. Clients and users may then use those tokens (JWT) with other Gen3 Data Commons services to access protected endpoints that require specific permissions. Fence can be configured to support different Identity Providers (IDPs) for AuthN. At the moment, supported IDPs include Google, and Shibboleth supporting providers such as NIH iTrust.
-### Guppy
-Server that support GraphQL queries on data from elasticsearch.
-### Hatchery
-Hatchery creates Kubernetes Pods for workspace services. Workspace services must expose HTTP servers. Ambassador is used to proxy user traffic through to their container workspace once it is launched by Hatchery.
-### Helm
-Gen3 relies upon Helm to manage installation and management of Kubernetes applications. Helm is used to build ”charts”, which are packages of Kubernetes resources that are used to deploy apps to a cluster. Helm is the recommended way to deploy Gen3.
-### Metadata Service (MDS)
-The Metadata Service provides an API for retrieving JSON metadata of GUIDs. It is a flexible option for "semi-structured" data (key:value mappings).  The content of the MDS powers the Data Portal Discovery Page for a Data Commons. The Gen3 SDK can be used to upload and edit the metadata.
-### Pelican
-Provides Docker images with Sower jobs to export and import PFB in Gen3.
-### Peregrine
-Peregrine is the high speed metadata seeking service which responds to GraphQL search queries. The GraphQL service allows Commons operators and users to precisely query only the information they are most interested in from the metadata collections. The service translates the GraphQL search into the appropriate statements which are run against the PostgreSQL backend before being returned as friendly JSON.
-### Sheepdog
-The Sheepdog service is responsible for herding user submissions of metadata into the graph database. The submissions are quality controlled against the data dictionary to ensure all required fields are present and have appropriate data values. The Sheepdog service is also responsible for supporting bulk export of the metadata into TSV or JSON formats.
-### Sower
-Sower dispatches Kubernetes jobs.
-### Tube
-Microservice that controls the ETL process of structured data.
-### Workspace Token Service
-The Gen3 workspace token service acts as an OIDC client which acts on behalf of users to request refresh tokens from Fence. This happens when a user logs into a workspace from the browser. WTS then stores the refresh token for that user, and manages access tokens and refresh tokens for workers that belong to specific users in the workspace.
+A simple list of most relevant microservices are included below.  For a description of each service and links to their respective repositories please visit the [Developer's Guide][Microservices].
+
+* Aggregated Metadata Service (AggMDS)
```

Comment:
This isn't a separate service, so I wouldn't include it in this list. This case is actually what makes it difficult to describe our offerings as services rather than APIs. There's only 1 service, the MDS, which exposes the MDS API and the aggMDS API.

Reply:
re: this -- although it's technically not a separate service, i think we think about and talk about it as though it is a separate  service. especially because they have the 2 endpoints. maybe we can talk more about that in a asterisky way -- but it feels like something that should be treated differently, even if technically it isn't

Reply:
Notably -- arguing more for you -- there is only one Helm chart (called metadata) that covers both of these

Reply:
Upon further discussion, we will remove the AggMDS repo since the code governing the AggMDS is actually all within the MDS repo.